extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -764,7 +764,8 @@ export class AmpA4A extends AMP.BaseElement {
               return true;
             });
         } catch (e) {
-          dev().error('Error injecting creative in friendly frame', e);
+          dev().error(
+            'Amp Ad', 'Error injecting creative in friendly frame', e);
```

Comment:
Undo this one, I've already got a [fix](https://github.com/ampproject/amphtml/pull/6005) for it.